ts/src/profiler.ts

```diff
@@ -72,6 +73,21 @@ export interface RequestProfile {
   labels?: {instance?: string};
 }
 
+interface ServerBackoffResponse {
+  statusMessage: string;
+  body: {details: {retryDelay: string}};
+}
+
+/**
+ * Returns true if response indicates a backoff.
```

Comment:
Sometimes you say "Returns foo" and sometimes "@return foo". Is there a reason?

Reply:
I should use "@return foo". Switching to that, and added PR #78 to fix throughout.